TextToSpeech/TextToSpeechImplementation.cpp

```diff
@@ -191,6 +255,29 @@ namespace Plugin {
     uint32_t TextToSpeechImplementation::SetConfiguration(const Exchange::ITextToSpeech::Configuration &object,Exchange::ITextToSpeech::TTSErrorDetail &ttsStatus)
     {
         TTS::Configuration config;
```

Comment:
This declaration can be moved down (i.e only after the validation you might need the config instance)